runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -1687,6 +1687,76 @@ J9::SymbolReferenceTable::findOrCreateGlobalFragmentSymbolRef()
    }
 
 
+TR::SymbolReference *
+J9::SymbolReferenceTable::createKnownStaticReferenceSymbolRef(void *dataAddress, TR::KnownObjectTable::Index knownObjectIndex)
```

Comment:
This one was marked as not J9 specific in the OMR discussion - why is it here?

Reply:
Oh, I did not seem to have updated OpenJ9 after that. Will push new changes.